### PR TITLE
Add a django command to set up the demo data

### LINKED ISSUE #19: Create data population script
We should create a script that does the basic task of populating a local instance with some test data (workspaces, tables, networks). Creating this by hand is unnecessarily tedious, and is error prone.

### PR DESCRIPTION
**Pull request by JackWilb**
### Does this PR close any open issues?
Depends on #132 
Closes #19

### Give a longer description of what this PR addresses and why it's needed
This PR adds all the demo data into an instance of multinet from the command line. This gives us a standardized set of networks that can be set up on any multinet instance. I made all the workspaces public so it should be accessible from any client application without authentication.

### Provide pictures/videos of the behavior before and after these changes (optional)
image

### Are there any additional TODOs before this PR is ready to go?

TODOs:
- [x] Fix TODO in code, adding table column types

### CODE REVIEW

**Comment by AlmightyYakob** [comment]
What is the point of all the fetchdata / process scripts? Why not just check the "processed" data itself into the repo?

File: multinet/api/management/commands/setupdevenv.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+    def create_tables_for_workspace(self, workspace_name, edge_table_name):
+        try:
+            # Get the paths for all the data objects to upload
+            data_dir_path = Path(settings.BASE_DIR) / "data" / workspace_name
+            csv_paths = list(data_dir_path.glob("*.csv"))
+            workspace = Workspace.objects.get(name=workspace_name)
+
+            for csv_path in csv_paths:
+                filename = str(csv_path).split("/")[-1].split(".")[0]
```
Comment (AlmightyYakob) [inline multinet/api/management/commands/setupdevenv.py:51]: A more idiomatic way to do this

```suggestion
                filename = os.path.splitext(csv_path.name)[0]
```

File: multinet/api/management/commands/setupdevenv.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+                
+                # Check if table with name exists, if so delete
+                if Table.objects.filter(workspace=workspace, name=filename).exists():
+                    self.stdout.write(self.style.SUCCESS(
+                        f"{workspace_name}/{filename} already exists, removing to recreate"
+                    ))
+                    Table.objects.filter(workspace=workspace, name=filename).delete()
+
+                # Open the types file and read the types in
+                types_path = Path(str(csv_path).replace(f"{filename}.csv", "")) / "types" / f"{filename}.json"
```
Comment (AlmightyYakob) [inline multinet/api/management/commands/setupdevenv.py:61]:
```suggestion
                types_path = data_dir_path / "types" / f"{filename}.json"
```

File: multinet/api/management/commands/setupdevenv.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+    def add_arguments(self, parser):
+        parser.add_argument('email', type=str)
+
+    def handle(self, *args, **options):
+        # Check that owner exists
+        queried_users = User.objects.filter(email=options["email"])
+        if not queried_users.exists():
+            raise Exception('User did not exist')
+        self.user = queried_users.first()
```
Comment (AlmightyYakob) [inline multinet/api/management/commands/setupdevenv.py:122]:
```suggestion
        self.user = User.objects.filter(email=options["email"]).first()
        if self.user is None:
      	    raise Exception('User did not exist')        
```

File: multinet/api/management/commands/setupdevenv.py [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        self.create_tables_for_workspace("movies", "")
+        self.create_tables_for_workspace("openflights", "routes")
+
+        # Create the networks from the tables
+        self.create_network_in_workspace("boston", "membership", ["clubs", "members"])
+        self.create_network_in_workspace("eurovis-2019", "connections", ["people"])
+        self.create_network_in_workspace("miserables", "relationships", ["characters"])
+        # self.create_network_in_workspace("movies") Not possible with the data structure,
+        # maybe we could use the new network creation API from the command line
+        self.create_network_in_workspace("openflights", "routes", ["airports"])
```
Comment (AlmightyYakob) [inline multinet/api/management/commands/setupdevenv.py:144]:
```suggestion
        # Create the tables
        self.create_tables_for_workspace("boston", "membership")
        self.create_tables_for_workspace("eurovis-2019", "connections")
        self.create_tables_for_workspace("miserables", "relationships")
        self.create_tables_for_workspace("openflights", "routes")

        # Create the networks from the tables
        self.create_network_in_workspace("boston", "membership", ["clubs", "members"])
        self.create_network_in_workspace("eurovis-2019", "connections", ["people"])
        self.create_network_in_workspace("miserables", "relationships", ["characters"])
        self.create_network_in_workspace("openflights", "routes", ["airports"])
        
        # TODO: Include movies dataset here once new network creation API is accessible internally from the API
```

**Review by AlmightyYakob — CHANGES_REQUESTED** [review]
Looks good overall, just some code suggestions.

**Comment by AlmightyYakob** [comment]
@JackWilb can we also remove all of the `scripts` folders? If keeping track of the data source is important, I'd think we could just put some kind of `README` in the `data` folder to illustrate that. My main gripe with what's there now is that there's lots of these `script` folders, with no explanation of why they're there / what role they play in the system.

**Comment by JackWilb** [comment]
> @JackWilb can we also remove all of the `scripts` folders? If keeping track of the data source is important, I'd think we could just put some kind of `README` in the `data` folder to illustrate that. My main gripe with what's there now is that there's lots of these `script` folders, with no explanation of why they're there / what role they play in the system.

Roni and I discussed this a little on slack. The scripts are good for showing a user how to transform data from a raw format to one that will work with our csv uploaders. I think that removing them entirely would remove some value that we get from them. Do you think it's okay to keep them here and link to them from the docs, or should we consider moving them to another place?

**Comment by JackWilb** [comment]
Force pushed to rebase on #132 updates, which will also need a review before this can be merged

**Review by AlmightyYakob — APPROVED** [review]

**Pull request merged**